cross-project-tests/debuginfo-tests/dexter/dex/debugger/Debuggers.py

```diff
@@ -47,7 +47,7 @@ def _get_potential_debuggers():  # noqa
 
 
 def _warn_meaningless_option(context, option):
-    if hasattr(context.options, "list_debuggers"):
+    if hasattr(context.options, "list-debuggers"):
```

Comment:
I'd have thought `list-debuggers` isn't a valid variable name, I'm confused how this works?